src/radical/pilot/agent/agent_0.py

```diff
@@ -784,14 +784,12 @@ def _check_tasks_cb(self):
     #
     def _prepare_env(self, env_name, env_spec):
 
-        print(env_spec)
-
         etype = env_spec['type']
         evers = env_spec['version']
         emods = env_spec.get('setup')    or []
         pre   = env_spec.get('pre_exec') or []
 
-        pre_exec = '-P ". env/bs0_orig.sh" '
+        pre_exec = '-P ". env/bs0_pre_0.sh" '
```

Comment:
not clear with this, which script is assumed to be changed `bs0_orig.sh` or `bs0_pre_0.sh`? (I understand that at this point they are the same)

Reply:
I'm positive on this one - changes need to apply to the pre_bootstrap_0 also (which results in the `bs0_pre_0` env).  Thanks for checking!